Form-Controls/index.html

```diff
@@ -13,16 +13,75 @@
     <h1>Product Pick</h1>
   </header>
   <main>
-  <form>
+  <form class="form" action="reesults.html" method="GET">
     <!-- write your html here-->
     <!-- try writing out the requirements first-->
-  </form>
+     <div>
+        <label for="name">Name</label>
+        <input type="text" id="name" required>
+     </div>
+     <div>
+        <label for="email">Email</label>
+        <input type="email" id="email" required>
+     </div>
+     
+     <div>
+      <label for="date">Delivery date (within the next 4 weeks):</label>
+      <input type="date" id="date" >
+     </div>
+     Colour
```

Comment:
Do you think I can submit a T-shirt without choosing colour? Do you think choosing colour is required?